### PR TITLE
Update travis and toolchain script

### PR DESCRIPTION
**Pull request by AnthonyAmanse**
Updated scripts to reflect secrets configuration

### CODE REVIEW

File: scripts/deploy-to-bluemix/deploy.sh [hunk 1]
```diff
@@ -67,25 +67,30 @@ if [[ -z $MYSQL_DB_USER ]] && [[ -z $MYSQL_DB_PASSWORD ]] && [[ -z $MYSQL_DB_HOS
...
+  export USERNAME_BASE64=$(echo -n book_user | base64)
+  export PASSWORD_BASE64=$(echo -n password | base64)
+  export HOST_BASE64=$(echo -n book-database | base64)
+  export PORT_BASE64=$(echo -n 3306 | base64)
+
+  sed -i s#"YWRtaW4="#$USERNAME_BASE64#g secrets.yaml
+  sed -i s#"VEhYTktMUFFTWE9BQ1JPRA=="#$PASSWORD_BASE64#g secrets.yaml
+  sed -i s#"c2wtdXMtc291dGgtMS1wb3J0YWwuMy5kYmxheWVyLmNvbQ=="#$HOST_BASE64#g secrets.yaml
+  sed -i s#"MTg0ODE="#$PORT_BASE64#g secrets.yaml
+  cat secrets.yaml
```
Comment (Tomcli) [inline scripts/deploy-to-bluemix/deploy.sh]: Hmm where do you apply the `secrets.yaml`?

File: scripts/install.sh [hunk 2]
```diff
@@ -26,10 +26,15 @@ function cluster_setup() {
...
-sed -i s#PLACEHOLDER_DB_PORT#3306#g $(ls | grep new)
+export USERNAME_BASE64=$(echo -n book_user | base64)
+export PASSWORD_BASE64=$(echo -n password | base64)
+export HOST_BASE64=$(echo -n book-database | base64)
+export PORT_BASE64=$(echo -n 3306 | base64)
+
+sed -i s#"YWRtaW4="#$USERNAME_BASE64#g secrets.yaml
+sed -i s#"VEhYTktMUFFTWE9BQ1JPRA=="#$PASSWORD_BASE64#g secrets.yaml
+sed -i s#"c2wtdXMtc291dGgtMS1wb3J0YWwuMy5kYmxheWVyLmNvbQ=="#$HOST_BASE64#g secrets.yaml
+sed -i s#"MTg0ODE="#$PORT_BASE64#g secrets.yaml
```
Comment (Tomcli) [inline scripts/install.sh]: Same here, I don't see you apply the secrets.yaml

File: scripts/deploy-to-bluemix/deploy.sh (hunk 1 above)
Comment (AnthonyAmanse) [inline scripts/deploy-to-bluemix/deploy.sh]: Added. Thanks!

File: scripts/install.sh (hunk 2 above)
Comment (AnthonyAmanse) [inline scripts/install.sh]: Added. Thanks!

**Comment by Tomcli** [comment]
LGTM

**Pull request merged**